### PR TITLE
Kuudra Part 2

### PR DESCRIPTION
**Pull request by AzureAaron**
I've added a kuudra glow and a danger warning, I plan to make more but it will take longer so I'd rather people be able to enjoy these new features for the release.

### CODE REVIEW

**Comment by Fluboxer** [comment]
There are also few popular features on 1.8 mods regarding announcing fresh tools, highlighting teammates with it (parse chat) and showing where to go next for optimal building

File: src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/KuudraGlow.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package de.hysky.skyblocker.skyblock.crimson.kuudra;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import net.minecraft.entity.mob.MagmaCubeEntity;
+
+public class KuudraGlow {
```
Comment (viciscat) [inline src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/KuudraGlow.java:6]: should this be a class just for one static method and int?

File: src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/DangerWarning.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+	private static void updateIndicator() {
+		if (Utils.isInKuudra() && SkyblockerConfigManager.get().crimsonIsle.kuudra.dangerWarning && CLIENT.player != null && CLIENT.world != null) {
+			BlockPos under = CLIENT.player.getBlockPos().down();
+			BlockPos under2 = CLIENT.player.getBlockPos().down(2);
+			BlockPos under3 = CLIENT.player.getBlockPos().down(3);
+			BlockPos under4 = CLIENT.player.getBlockPos().down(4);
+			BlockPos under5 = CLIENT.player.getBlockPos().down(5);
+
+			Title title = ObjectUtils.firstNonNull(getDangerTitle(under), getDangerTitle(under2), getDangerTitle(under3), getDangerTitle(under4), getDangerTitle(under5));
```
Comment (viciscat) [inline src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/DangerWarning.java:41]: for loop? 😭

**Review by viciscat — COMMENTED** [review]
Unrelated but why do we have 2 methods for MobGlow (shouldGlow and glowColor)? Can't we just make glowColor return an OptionalInt? Sometimes both methods call the same method

File: src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/DangerWarning.java (hunk 2 above)
Comment (AzureAaron) [reply src/main/java/de/hysky/skyblocker/skyblock/crimson/kuudra/DangerWarning.java]: I completely forgot these existed when I wrote this lmfao, was very late at night :c

**Review by viciscat — APPROVED** [review]
my unrelated remark got ignored D:

**Comment by AzureAaron** [comment]
> my unrelated remark got ignored D:

We could perhaps change it to work like this but after the slayer entity glow and maybe my carnival stuff happens since those also touch that part of the code

**Pull request merged**